bin/cml/comment/create.js

```diff
@@ -55,5 +55,12 @@ exports.options = kebabcaseKeys({
     type: 'boolean',
     description:
       'Avoid watermark; CML needs a watermark to be able to distinguish CML comments from others'
+  },
+  commentLabel: {
```

Comment:
to avoid duplication (`cml comment ... --comment`)

```suggestion
  id: {
```

Reply:
how about `label`?

Reply:
unfortunately that overloads/conflicts with:

- [CML runner labels](https://cml.dev/doc/ref/runner#--labels)
- [GH labels](https://docs.github.com/en/issues/using-labels-and-milestones-to-track-work/managing-labels)

Reply:
`tag`?

Reply:
at the risk of being to verbose, I would learn towards `watermarkId` or `watermarkTag`. Certainly not plain `id` though?

Reply:
`watermark` is arguably an internal cml concept, that should have limited exposure to users - `commentTag`?

Reply:
`commentTag` 👍 lets go with that.

Reply:
not a fan (clashes with Git tags).

`--id` seems best UX to me. It doesn't clash with anything, at least as far as end-users are concerned.

Reply:
Let's take a step back and see which command line makes more sense:
```
cml comment create --update --pr --publish --comment-label='{run}-best-result'
```
or 
```
cml comment create --update --pr --publish --id='{run}-best-result'
```

Reply:
the group has decided on `--cml-comment-id`